buildSrc/build.gradle.kts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2024 ForgeRock. All rights reserved.
+ *
+ *  This software may be modified and distributed under the terms
+ *  of the MIT license. See the LICENSE file for details.
+ */
+
+plugins {
+    `kotlin-dsl`
+}
+
+repositories {
```

Comment:
i think having the repositories declared in the settings.gradle.... do we need here ? - https://developer.android.com/build#settings-file

Reply:
Yes, we need this for buildSrc, when I removed it, the build failed.